Automattic/jetpack
# VideoPress: fix potential use of unexisting allow_download property

#### Changes proposed in this Pull Request:

This PR adds a check to verify the allow_download property exists when displaying the "allow download" checkbox on the media modal.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* On a sandboxed jetpack site with VideoPress enabled
* `jetpack docker tail`
* Go to /wp-admin/upload.php
* Upload a video on VideoPress or select an existing one that doesn't have the "allow download" option enabled
* ✅ No warning or error should be printed in the docker tail output
